web/examples/craft/main.dart

```diff
@@ -67,9 +68,14 @@ void main() {
 }
 
 void startCraft() {
+  int seed = _getSeed();
+  if (seed == null) {
```

Comment:
Relating to [this comment](https://github.com/Grant-Nelson/ThreeDart/pull/74/files#r225640223), I would recommend,
```suggestion
  if (seed < 0) {
```